charts/incubator/torrserver/values.yaml

```
@@ -0,0 +1,46 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: smailkoz/torrserver
+  # -- image tag
+  tag: 1.0.111
```

```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```

In new charts we are trying to define the appVersion only once, that way we "force" ourselves to update the appVersion in Chart.yaml as well